traitcuration/traits/models.py

```diff
@@ -37,13 +41,20 @@ def __str__(self):
         return self.name
 
 
-class User(models.Model):
-    username = models.CharField(max_length=100, unique=True)
-    email = models.EmailField(unique=True)
-    role = models.CharField(max_length=20, blank=True, null=True)
```

Comment:
Looks like we lost the "role" field in the refactoring. Or is it included in some form in the AbstractUser and inherited from there?

Reply:
You are right, I wll add that.